lib/paper_trail/events/create.rb

```diff
@@ -13,7 +13,8 @@ class Create < Base
       # @api private
       def data
         data = {
-          item: @record,
+          item_id: record_id,
+          item_type: @record.class.base_class.name,
```

Comment:
If we are going to specify `item_type`, we need to be careful about STI. Do our tests cover STI, ie. do we have assertions about the persisted `item_type`?

Reply:
I took this implementation from https://github.com/paper-trail-gem/paper_trail/blob/master/lib/paper_trail/events/destroy.rb#L17 and I think it works fine with STI. I can add some tests just in case